app/views/checkout/payment.scala.html

```diff
@@ -1,95 +1,119 @@
+                            <input type="hidden" class="js-input" name="personal.emailValidation.confirm" id="confirm" value="@form.data.get(" personal.emailValidation.confirm ")"> }
                         </div>
                         <div class="js-checkout-full-address">
                             <div class="form-field js-checkout-house">
                                 <label class="label" for="house">Address line 1</label>
-                                <input type="text" class="input-text js-input" name="personal.address.address1" value="@form.data.get("personal.address.address1")">
-                                @checkout.errorMessage("This field is required")
+                                <input type="text" class="input-text js-input" name="personal.address.address1" value="@form.data.get(" personal.address.address1 ")"> @checkout.errorMessage("This field is required")
                             </div>
                             <div class="form-field js-checkout-street">
                                 <label class="label" for="street">Address line 2</label>
-                                <input type="text" class="input-text js-input" name="personal.address.address2" value="@form.data.get("personal.address.address2")">
-                                @checkout.errorMessage("This field is required")
+                                <input type="text" class="input-text js-input" name="personal.address.address2" value="@form.data.get(" personal.address.address2 ")"> @checkout.errorMessage("This field is required")
                             </div>
                             <div class="form-field js-checkout-town">
                                 <label class="label" for="town">Town/City</label>
-                                <input type="text" class="input-text js-input" name="personal.address.town" value="@form.data.get("personal.address.town")">
-                                @checkout.errorMessage("This field is required")
+                                <input type="text" class="input-text js-input" name="personal.address.town" value="@form.data.get(" personal.address.town ")"> @checkout.errorMessage("This field is required")
                             </div>
                         </div>
                         <div class="form-field js-checkout-postcode">
                             <label class="label" for="postcode">Postcode</label>
-                            <input type="text" class="input-text js-input input-text input-text--small" name="personal.address.postcode" value="@form.data.get("personal.address.postcode")">
-                            @checkout.errorMessage("Please enter a valid postal code")
+                            <input type="text" class="input-text js-input input-text input-text--small" name="personal.address.postcode" value="@form.data.get(" personal.address.postcode ")"> @checkout.errorMessage("Please enter a valid postal code")
```

Comment:
I think the form might not pre-populate the fields because `form.data.get` argument has whitespace padding. Or is it still working?